examples/todo/ui/todo.slint

```diff
@@ -69,6 +76,28 @@ MainWindow := Window {
             }
         }
 
+        if (is-header-visible) : HorizontalBox { 
+            alignment: start; 
+
+            CheckBox {  
+                text: "Sort by name";
+                checked <=> is-sort-by-name;
+
+                toggled => {
+                    sort_filter();
+                }
+            }
+
+            CheckBox {  
+                text: "Hide done items";
+                checked <=> is-filter-done;
+
+                toggled => {
+                    sort_filter();
```

Comment:
The name of the callbacks might make innocent readers think "ah, this sorts the filter - I wonder what that means". I suggest to rename the callback to "apply_sorting_and_filtering()".